Libraries/LibLine/Editor.cpp

```diff
@@ -921,6 +921,17 @@ void Editor::handle_read_event()
             }
             continue;
         }
+        // ^T
```

Comment:
^Today I learned about this

Reply:
why are these puns all over the place ^today?